playground/fs-serve/root/src/index.html

```diff
@@ -53,8 +53,10 @@ <h2>Denied</h2>
   text('.full', JSON.stringify(json))
   text('.named', msg)
 
+  const base = typeof BASE !== 'undefined' ? BASE : ''
+
   // inside allowed dir, safe fetch
-  fetch('/src/safe.txt')
+  fetch(base + '/src/safe.txt')
```

Comment:
if we've got `/base/` then we'll end up with a `//`. This should probably use `joinUrlSegments` or `path.posix.join`

Reply:
i use `joinUrlSegments ` to fix that